src/client/services/signalR.service.js

```diff
@@ -23,6 +23,10 @@ export const disconnect = () => {
   connection.stop();
 };
 
+export const emit = (event, ...args) => {
+  connection.invoke(event, ...args);
```

Comment:
I think you should use the `send` method here because it doesn't wait for the server to process the event (link `emit` in `socket-io`)